### PR TITLE
Update privacy notice for EOL (EXPOSUREAPP-14955)

### PR DESCRIPTION
**Pull request by SamuraiKek**
[Ticket.](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14955)

Please compare the contents of the word documents found in the ticket with the changes made to the privacy files.

### CODE REVIEW

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 1]
```diff
@@ -25,28 +133,25 @@
...
 </p>
 <p>
-    <strong>12. Wie können Sie Ihr Einverständnis zurücknehmen?</strong>
+    <strong>12. Welche weiteren Datenschutzrechte haben Sie?</strong>
 </p>
 <p>
-    <strong>13. Welche weiteren Datenschutzrechte haben Sie?</strong>
-</p>
-<p>
-    <strong>14. Datenschutzbeauftragte und Kontakt</strong>
```
Comment (SamuraiKek) [inline Corona-Warn-App/src/main/assets/privacy_de.html:49]: The word docs say there are 14 points, but number 7 got deleted. Only 13 points remain.

**Review by CV113 — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/assets/privacy_tr.html [hunk 2]
```diff
@@ -1,10 +1,112 @@
...
+    <strong>2. Verileriniz kime aktarılacak?</strong>
+</p>
+<p>
+    1 Mayıs 2023 tarihinden itibaren RKI, Corona-Warn-App uygulamasının çalışması için artık kişisel
+    verileri işlemeyecektir. Boş modda, Corona-Warn-App artık RKI'nin sunucu sistemine veya diğer
+    alıcılara veri aktarmayacaktır. Corona-Warn-App yalnızca akıllı telefonunuzdaki verilerinizi
+    (temas günlüğündeki girişler ve kayıtlı COVID sertifikaları) işler.
+</p>
+<p>
+    <strong>3. Wie können Sie Ihr Einverständnis zurücknehmen?</strong>
```
Comment (OLVI5) [inline Corona-Warn-App/src/main/assets/privacy_tr.html:68]: here are some german heading - translation should be like this (checked with a turkish speaking colleague) 

3 Onayınızı nasıl geri çekebilirsiniz?

Comment (SamuraiKek) [reply Corona-Warn-App/src/main/assets/privacy_tr.html:68]: Ok

File: Corona-Warn-App/src/main/assets/privacy_en.html [hunk 3]
```diff
@@ -25,28 +128,25 @@
...
-    <strong>11. Is your data transferred to countries outside the EU?</strong>
+    <strong>11. How can you withdraw your consent?</strong>
 </p>
 <p>
-    <strong>12. How can you withdraw your consent?</strong>
+    <strong>12. What other rights do you have under data protection law?</strong>
 </p>
 <p>
-    <strong>13. What other rights do you have under data protection law?</strong>
-</p>
```
Comment (OLVI5) [inline Corona-Warn-App/src/main/assets/privacy_en.html:47]: the document contains of 14 chapters and the chapter 13 is 
13. What other rights do you have under data protection law?

Comment (OLVI5) [reply Corona-Warn-App/src/main/assets/privacy_en.html:47]: all correct i was wrong

**Review by OLVI5 — DISMISSED** [review]

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 4]
```diff
@@ -1,9 +1,117 @@
+<h1>
+    Verarbeitung personenbezogener Daten
+</h1>
```
[the remark this answers is not in the document]
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/assets/privacy_de.html:3]: Yes, you're right. Updated the heading.

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 5]
```diff
@@ -1,9 +1,117 @@
...
+    per E-Mail an: <a href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
+</p>
+
+<hr>
+
+<h1>
+    Datenschutzerklärung
+</h1>
+<p>
+    Version 3.20 (gültig bis 30.04.2023)
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/assets/privacy_de.html:110]:
```suggestion
    Version 3.2.0 (gültig bis 30.04.2023)
```

File: Corona-Warn-App/src/main/assets/privacy_en.html [hunk 6]
```diff
@@ -1,9 +1,112 @@
...
+    the RKI’s official data protection officer by post to: Robert Koch-Institut, FAO the data
+    protection officer, Nordufer 20, 13353 Berlin, or by emailing: <a
+        href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
 </p>
+
+<hr>
+
+<h1>Privacy notice</h1>
 <p>
+    Version 3.20 (valid until 30 April 2023)
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/assets/privacy_en.html:107]:
```suggestion
    Version 3.2.0 (valid until 30 April 2023)
```

File: Corona-Warn-App/src/main/assets/privacy_tr.html [hunk 7]
```diff
@@ -1,10 +1,112 @@
...
+    görevlisi), Nordufer 20, 13353 Berlin veya e-posta yoluyla: <a href="mailto:datenschutz@rki.de">datenschutz@rki.de</a>.
+</p>
+
+<hr>
+
+<h1>
+    Veri Gizliliği Beyanı
+</h1>
+<p>
+    Sürüm 3.20 (30.04.2023 tarihine kadar geçerlidir)
```
Comment (Ein-Tim) [inline Corona-Warn-App/src/main/assets/privacy_tr.html:105]:
```suggestion
    Sürüm 3.2.0 (30.04.2023 tarihine kadar geçerlidir)
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Attention! The version number is wrong!

File: Corona-Warn-App/src/main/assets/privacy_de.html (hunk 5 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/assets/privacy_de.html:110]: Good catch!

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 8]
```diff
@@ -1,9 +1,117 @@
...
+    installiert ist. Sie können jedoch keine neuen COVID-Zertifikate in der App speichern und
+    erhaltenen keine Hinweise zur Erneuerung von COVID-Zertifikaten. Wenn Sie die gespeicherten
+    COVID-Zertifikate und das Kontakt-Tagebuch weiterhin verwenden wollen, beachten Sie bitte die
+    Hinweise am Ende dieses Abschnitts.
+    Mit Aktivierung des Ruhemodus wird die Corona-Warn-App das COVID-19-Benachrichtigungssystem
+    Ihres Betriebssystems, mit dem die Begegnungsdaten ausgesendet und Begegnungen aufgezeichnet
+    werden (iPhone: „Begegnungsmitteilungen“; Android-Smartphones: „COVID-19-Benachrichtigungen“)
+    deaktivieren. Sie können das COVID-19-Benachrichtigungssystem jederzeit auch selbst in den
+    Einstellungen Ihres Betriebssystems deaktivieren. Ihr Betriebssystem löscht die auf ihrem
+    Smartphone gespeicherten Begegnungsdaten nach 14 Tagen automatisch. Falls Sie eine frühere
```
Comment (MikeMcC399) [inline Corona-Warn-App/src/main/assets/privacy_de.html:32]: Typo:

"Ihr Betriebssystem löscht die auf ihrem Smartphone gespeicherten Begegnungsdaten nach 14 Tagen automatisch."

should be auf "Ihrem Smartphone"

File: Corona-Warn-App/src/main/assets/privacy_en.html [hunk 9]
```diff
@@ -1,9 +1,112 @@
...
+    <u>
+        Notes on using the features for digital COVID certificates and the contact journal from 1
+        May 2023:
+    </u>
+</p>
+<ul>
+    <li>
+        You can still use the export features to back up your stored digital COVID certificates and
+        entries in your contact journal. If you wish to present a digital COVID certificate as
+        proof, please note the information in Section Error: Reference source not found of the
```
Comment (MikeMcC399) [inline Corona-Warn-App/src/main/assets/privacy_en.html:47]: "Section Error: Reference source not found"

is an error

Comment (Ein-Tim) [reply Corona-Warn-App/src/main/assets/privacy_en.html:47]: @OLVI5

There are multiple issues with the updated texts. Please have a look again at both the iOS PR and the Android PR and make sure that all issues are fixed, also in the PDF that will go live on the website.

Comment (OLVI5) [reply Corona-Warn-App/src/main/assets/privacy_en.html:47]: Hi @Ein-Tim ,
we will look at the proposed changes and go through the documents again. Thank you very much!
Best Oliver

**Comment by MikeMcC399** [comment]
It would be good to have a new open PR to make comments on the text contents.

File: Corona-Warn-App/src/main/assets/privacy_de.html [hunk 10]
```diff
@@ -671,7 +771,7 @@ <h3>
 <p>
     Die Risiko-Ermittlung ist eine Hauptfunktion der App. Sie dient dazu, Sie
     bei möglichen Begegnungen mit Corona-positiv getesteten Personen
-    (Risiko-Begegnungen) länderübergreifend zu warnen, Ihr persönliches
```
[the remark this answers is not in the document]
Comment (SamuraiKek) [reply Corona-Warn-App/src/main/assets/privacy_de.html:674]: The continuation is below: `Ihr persönliches Infektionsrisiko zu bewerten und Ihnen Verhaltens- und Gesundheitshinweise zu geben.`

**Pull request merged**